Bembel/src/HomogenisedLaplace/SingleLayerPotential.hpp

```diff
@@ -0,0 +1,97 @@
+// This file is part of Bembel, the higher order C++ boundary element library.
+//
+// Copyright (C) 2022 see <http://www.bembel.eu>
+//
+// It was written as part of a cooperation of J. Doelz, H. Harbrecht, S. Kurz,
+// M. Multerer, S. Schoeps, and F. Wolf at Technische Universitaet Darmstadt,
+// Universitaet Basel, and Universita della Svizzera italiana, Lugano. This
+// source code is subject to the GNU General Public License version 3 and
+// provided WITHOUT ANY WARRANTY, see <http://www.bembel.eu> for further
+// information.
+
+#ifndef BEMBEL_SRC_HOMOGENISEDLAPLACE_SINGLELAYERPOTENTIAL_HPP_
+#define BEMBEL_SRC_HOMOGENISEDLAPLACE_SINGLELAYERPOTENTIAL_HPP_
+
+namespace Bembel {
+// forward declaration of class HomogenisedLaplaceSingleLayerPotential
+// in order to define traits
+template<typename LinOp>
+class HomogenisedLaplaceSingleLayerPotential;
+
+template<typename LinOp>
+struct PotentialTraits<HomogenisedLaplaceSingleLayerPotential<LinOp>> {
+  typedef Eigen::VectorXd::Scalar Scalar;
+  static constexpr int OutputSpaceDimension = 1;
+};
+
+/**
+ * \ingroup HomogenisedLaplace
+ */
+template<typename LinOp>
+class HomogenisedLaplaceSingleLayerPotential: public PotentialBase<
+    HomogenisedLaplaceSingleLayerPotential<LinOp>, LinOp> {
+  // implementation of the kernel evaluation, which may be based on the
+  // information available from the superSpace
+
+ private:
```

Comment:
(optional) private members are at the bottom of the class definition in Bembel.

Reply:
Done